Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java

```diff
@@ -130,11 +113,45 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     {
       mBinding.textGameTitle.setText(title);
     }
+  }
 
-    if (getResources().getConfiguration().getLayoutDirection() == View.LAYOUT_DIRECTION_LTR)
+  private void setInsets()
+  {
+    ViewCompat.setOnApplyWindowInsetsListener(mBinding.getRoot(), (v, windowInsets) ->
     {
-      view.post(() -> NativeLibrary.SetObscuredPixelsLeft(view.getWidth()));
-    }
+      Insets cutInsets = windowInsets.getInsets(WindowInsetsCompat.Type.displayCutout());
+      mCutInset = cutInsets.left;
+
+      int left = 0;
+      int right = 0;
+      if (ViewCompat.getLayoutDirection(v) == ViewCompat.LAYOUT_DIRECTION_LTR)
+      {
+        left = cutInsets.left;
+      }
+      else
+      {
+        right = cutInsets.right;
+      }
+
+      v.post(() ->
+      {
+        NativeLibrary.SetObscuredPixelsTop(cutInsets.top);
```

Comment:
If you're not going to reset ObscuredPixelsTop at any point, you should probably be setting it in EmulationActivity's setInsets instead of when opening the menu.